app/views/customers/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Sign up</h1>
+
+<%= form_for(@customer) do |f| %>
+    <%= f.label :name %><br>
+    <%= f.text_field :name %><br>
+    <%= f.label :email %><br>
+    <%= f.email_field :email %><br>
+    <%= f.label :password %><br>
+    <%= f.password_field :password %><br>
+    <%= f.submit "Sign up" %>
+<% end %>
```

Comment:
here's why github is showing you this red sign: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline